style.css

```diff
@@ -0,0 +1,3 @@
+header {
+  color: #ff7b00;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly this color is not working, please fix it 🙏🏼 In my opinion remove the `header` tag and try to add style `h1` tag 😃 

![image](https://user-images.githubusercontent.com/63915024/219333887-4626acda-c595-468c-b0aa-d2857b043510.png)

Reply:
I have made this correct, thank you